packages/runtime/src/runtime/screen/wrapper.tsx

```diff
@@ -0,0 +1,43 @@
+import type { EnsembleAPIModel } from "@ensembleui/react-framework";
+import { useScreenData } from "@ensembleui/react-framework";
+import { useEffect } from "react";
+import { useEnsembleAction } from "../hooks";
+
+export const ScreenApiWrapper: React.FC<{ children: React.ReactNode }> = ({
+  children,
+}) => {
+  const { apis = [], setApi } = useScreenData();
+
+  return (
+    <>
+      {apis.map((api, index) => (
+        <EvaluateApi
+          api={api}
+          currentIndex={index}
```

Comment:
You don't need index. Each API in the screen is uniquely identified by its name.